src/configure/configure.ts

```diff
@@ -166,7 +166,9 @@ class PipelineConfigurer {
             await this.createGithubServiceConnection();
         }
 
-        await this.createAzureRMServiceConnection();
+        if(this.inputs.pipelineParameters.pipelineTemplate.targetType != TargetResourceType.None && !!this.inputs.pipelineParameters.pipelineTemplate.targetKind) {
```

Comment:
I didn't understand why do we need to check for both target type and kind ? Will there be a scenario where target type is none but kind is not ? Please reconsider if we can just go ahead with only check for target type as none.

Reply:
I've updated